.github/CODEOWNERS

```diff
@@ -237,7 +237,7 @@ packages/react-components/react-storybook-addon-export-to-sandbox @microsoft/flu
 packages/react-components/babel-preset-storybook-full-source @microsoft/fluentui-react-build
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30553&commitId=02b925bc14b5d0417874d3c368bb1212e6ddd8c0&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Menu Example Divider Horizontal.chromium.png|76| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>

Reply:
I don't really see any noticeable difference except for maybe some pixel opacity/kerning in the font?